schema/deploy/tables/funding_parameter.sql

```diff
@@ -0,0 +1,46 @@
+-- Deploy cif:tables/funding_parameter to pg
+
+begin;
+
+create table cif.funding_parameter(
+  id integer primary key generated always as identity,
+  project_id integer references cif.project(id) not null,
+  total_project_value numeric(20,2),
+  max_funding_amount numeric(20,2),
+  province_share_percentage numeric,
+  holdback_percentage numeric,
+  anticipated_funding_amount numeric(20,2)
+);
+
+select cif_private.upsert_timestamp_columns('cif', 'funding_parameter');
+
+do
+$grant$
+begin
+
+-- Grant cif_internal permissions
+perform cif_private.grant_permissions('select', 'funding_parameter', 'cif_internal');
+perform cif_private.grant_permissions('insert', 'funding_parameter', 'cif_internal');
+perform cif_private.grant_permissions('update', 'funding_parameter', 'cif_internal');
+
+-- Grant cif_admin permissions
+perform cif_private.grant_permissions('select', 'funding_parameter', 'cif_admin');
+perform cif_private.grant_permissions('insert', 'funding_parameter', 'cif_admin');
+perform cif_private.grant_permissions('update', 'funding_parameter', 'cif_admin');
+
+-- Grant cif_external no permissions
+-- Grant cif_guest no permissions
+
+end
+$grant$;
+
+comment on table cif.funding_parameter is 'Table containing budget related data';
+comment on column cif.funding_parameter.id is 'Unique ID for the budget data';
+comment on column cif.funding_parameter.project_id is 'The related project for this data';
+comment on column cif.funding_parameter.total_project_value is 'Total project value';
+comment on column cif.funding_parameter.max_funding_amount is 'Maximum funding amount for this project';
+comment on column cif.funding_parameter.province_share_percentage is 'Province share percentage';
```

Comment:
When writing column comments, we want to be as explicit as possible. It may not be someone who is familiar with the project that is reading these (for example a DBA external to our team be the one looking at our metadata).

```suggestion
comment on column cif.funding_parameter.province_share_percentage is 'The percent of the total funding for a project that the Province is providing';
```